src/functionalTest/resources/features/F-002/S-022.td.json

```diff
@@ -0,0 +1,35 @@
+{
+  "title": "must receive an error response for an active caseworker who does not have document access",
+  "_guid_": "S-022",
+  "_extends_": "F-002_Test_Data_Base",
+
+  "specs": [
+    "contains an active caseworker who does not have document access"
+  ],
+
+  "request": {
+    "user": {
+      "username": "befta.caseworker.2.solicitor.1@gmail.com",
+      "password": "[[$CCD_BEFTA_CASEWORKER_2_SOLICITOR_1_PWD]]"
+    },
+    "pathVariables": {
+      "documentId": "${[scenarioContext][childContexts][Default_Document_Upload_Data][customValues][documentIdInTheResponse]}"
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_": "Common_401_Response",
+    "headers": {
+      "Connection" : "keep-alive",
+      "Content-Type" : "application/json",
+      "Keep-Alive" : "[[ANYTHING_PRESENT]]"
+    },
+    "body" : {
+      "timestamp" : "[[ANYTHING_PRESENT]]",
+      "status" : 401,
+      "error" : "Unauthorized",
```

Comment:
These 2 line looks redundant, as they are already inherited from Common_401_Response.

Reply:
Yes, it's inherited. Removed from response.